server/clcachesrv.py

```diff
@@ -114,9 +127,24 @@ def onSigterm(handle, signum):
 def main():
     logging.basicConfig(format='%(asctime)s [%(levelname)s]: %(message)s', level=logging.INFO)
 
+    parser = argparse.ArgumentParser(description='Process some integers.')
```

Comment:
The `description` looks odd, probably a leftover from copying this out of some example?